fv3core/stencils/c2l_ord.py

```diff
@@ -40,16 +40,15 @@ def c2l_ord2(
         va = a21 * u1 + a22 * v1
 
 
-@gtstencil()
 def ord4_transform(
     u: FloatField,
     v: FloatField,
+    dx: FloatFieldIJ,
```

Comment:
what was the reason to move those up? previously they were ordered alphabetically. I don't disagree with it, I just don't see it yet :wink:

Reply:
This is so that `c2l_ord2` and `ord4_transform` can be shared using the same frozen stencil (`_compute_cubed_to_latlon`) in init without having to make another logic to switch between the two.

Reply:
This is so that `ord4_transform` and `c2l_ord2` can share the same init frozen stencil name (`_compute_cubed_to_latlon`) without having another logic to switch between the two.

Reply:
great!